Sources/Public/Animation/AnimationView.swift

```diff
@@ -123,10 +125,24 @@ final public class AnimationView: AnimationViewBase {
   /**
    Describes the behavior of an AnimationView when the app is moved to the background.
 
-   The default is `pause` which pauses the animation when the application moves to
-   the background. The completion block is called with `false` for completed.
+   The default for the Main Thread animation engine is `pause`,
+   which pauses the animation when the application moves to
+   the background. This prevents the animation from consuming CPU
+   resources when not on-screen. The completion block is called with
+   `false` for completed.
+
+   The default for the Core Animation engine is `doNothing`, since
+   the Core Animation engine does not have any CPU overhead.
    */
-  public var backgroundBehavior: LottieBackgroundBehavior = .pause
+  public lazy var backgroundBehavior: LottieBackgroundBehavior = {
+    switch configuration.renderingEngine {
+    case .mainThread:
+      return .pause
+
+    case .coreAnimation:
+      return .doNothing
```

Comment:
We don't need to pause the Core Animation engine when it moves into the background, since it doesn't introduce any CPU overhead. This prevents us from having to `stop()` and then `start()` the animation, which avoids unnecessary animation setup passes.